src/components/Finance.jsx

```diff
@@ -7,30 +7,22 @@ import './dashboard.css';
 const Finance = () => {
 
     return (
-        <>
+        <section id="finance-table">
```

Comment:
This component should have it's own style sheet. It should not be using the dashboard styles.